packages/Molecules/Search/SearchResults.vue

```diff
@@ -34,14 +42,17 @@
 <script>
 import RplPagination from '@dpc-sdp/ripple-pagination'
 import RplSearchResult from './SearchResult.vue'
+import RplCardEvent from './../Card/CardEvent.vue'
```

Comment:
Hey @Ahmedjabar please import component out side this package by using package name. `import { RplCardEvent } from @dpc-sdp/ripple-card`. Otherwise it won't work in Nuxt, because the path is different in there.